packages/flutter/test/material/dropdown_form_field_test.dart

```diff
@@ -1382,4 +1382,34 @@ void main() {
     expect(find.text(forceErrorText), findsOne);
     expect(find.text(decorationErrorText), findsNothing);
   });
+
+  testWidgets('barrier Dismissible set to false', (WidgetTester tester) async {
```

Comment:
I think we should have another test which checks for dismiss behaviour with `true`. WDYT ?

Reply:
Name can be `Dropdown doeesn't close when barrier dismissible set to false`.

Reply:
That works. Although if we want to go a simpler approach I would simply test the default case, which would be `false`. But yes we should test it.